secant/UIComponents/CircularFrame/CircularFrameBadge.swift

```diff
@@ -114,6 +115,7 @@ struct BadgeOverlay: Animatable, ViewModifier {
                                 )
                                 .transition(.scale(scale: 2))
                                 .transition(.opacity)
+                                .shadow(color: Asset.Colors.Onboarding.badgeShadow.color, radius: 10, x: 0, y: 0)
```

Comment:
[non-blocking | nit | styling] Tiny nitpick, this line is a bit long ( > 100 chars), we could split it as below (totally up to you / preference though 😄 )

```suggestion
                                .shadow(
                                    color: Asset.Colors.Onboarding.badgeShadow.color,
                                    radius: 10,
                                    x: 0,
                                    y: 0
                                )
```